demo/meeting/src/containers/Navigation/index.tsx

```diff
@@ -22,21 +25,36 @@ const Navigation = () => {
   return (
     <Navbar className="nav" flexDirection="column" container>
       <NavbarHeader title="Navigation" onClose={closeNavbar} />
-      <NavbarItem
-        icon={<Attendees />}
-        onClick={toggleRoster}
-        label="Attendees"
-      />
-      <NavbarItem
-        icon={<Eye />}
-        onClick={toggleTheme}
-        label={theme === 'light' ? 'Dark mode' : 'Light mode'}
-      />
-      <NavbarItem
-        icon={<Information />}
-        onClick={toggleMetrics}
-        label="Meeting metrics"
-      />
+      <Flex
+        css={`
+          margin-top: 0rem;
+        `}
+      >
+        <NavbarItem
+          icon={<Attendees />}
+          onClick={toggleRoster}
+          label="Attendees"
+        />
+        <NavbarItem
+          icon={<Eye />}
+          onClick={toggleTheme}
+          label={theme === 'light' ? 'Dark mode' : 'Light mode'}
+        />
+        <NavbarItem
+          icon={<Information />}
+          onClick={toggleMetrics}
+          label="Bandwidth metrics"
+        />
+      </Flex>
+      <Flex marginTop="auto">
```

Comment:
Do we need flex in here? This can be just the same button that used to display bandwidth metrics right?